### DISCUSSION TITLE
Using MLKit with Swift Package manager

### DISCUSSION THREAD

**Discussion by Noor-Ahmed-Natali** [firebase/firebase-ios-sdk#9902] [Q&A]
I am using swift package manage for Firebase and i also want to implement face detection using ML kit I cannot find vision or face detection in firebase swift package manager. I tried it with using pods but as i am using package manager of all other stuff is shows compilation error when pods are installed

**Comment by paulb777** [comment]
@Noor-Ahmed-Natali Sorry, MLKit is no longer part of Firebase and MLKit does not currently provide Swift Package Manager support. There is a public discussion at https://github.com/googlesamples/mlkit/issues/180.

cc: @bcdj

**Comment by bpappin** [comment]
I haven't tried it yet, but this person is getting ahead of Google:
https://github.com/d-date/google-mlkit-swiftpm

**Comment by nodata** [comment]
Please add SPM support

**Comment by kmithi1-apps** [comment]
Please add SPM support

**Comment by brownsoo** [comment]
Please SPM & Simulator opt-out supports

**Comment by mustfaunlu** [comment]
add SPM support
